s3/s3.go

```diff
@@ -346,7 +353,7 @@ func NewFs(name, root string) (fs.Fs, error) {
 		c:                  c,
 		bucket:             bucket,
 		ses:                ses,
-		acl:                fs.ConfigFile.MustValue(name, "acl"),
+		acl:                fs.ConfigFile.MustValue(name, "acl", *ACL),
```

Comment:
This makes the config file override the command line argument - I think it should probably be the other way round.